docs/articles/documentation/reference/test-api/testcontroller/setnativedialoghandler.md

```diff
@@ -6,7 +6,7 @@ permalink: /documentation/reference/test-api/testcontroller/setnativedialoghandl
 # t.setNativeDialogHandler Method
 
 To handle native dialogs invoked during the test run, specify a handler function
-using the `setNativeDialogHandler` method of the
+with the `setNativeDialogHandler` method of the
 [test controller](README.md).
```

Comment:
Let's rewrite this and say:

`To handle native dialogs invoked during the test run, use the `setNativeDialogHandler` method of the [test controller](README.md) to specify a handler function.`

Reply:
I don't like the structure `**To** do this, use this **to** do this`

Can we say:

> To handle native dialogs invoked during the test run, specify a handler function with the setNativeDialogHandler method of the [test controller](README.md).

Reply:
I see, since these instructions are a little more complex (to the reader), we can go with your original suggestion. Thanks.